src/main/java/tatracker/logic/commands/CommandResult.java

```diff
@@ -31,13 +35,17 @@ public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
      * and other fields set to their default value.
      */
     public CommandResult(String feedbackToUser) {
-        this(feedbackToUser, false, false);
```

Comment:
Probably can replace the chain of booleans with a single enum!
The enum will decide what is the next action taken in the main window. Then ParserUtil can parse the next action, so that everything parsing related remains in parser!